documentation/document.css

```diff
@@ -130,6 +131,13 @@ html {
     margin: 0;
 }
 
+#code-iframe {
```

Comment:
Make it a class as you're using it in multiple places

Reply:
As Id has more specificity , I need to use it. If I use class the properties are not getting applied